docker-compose.yaml

```diff
@@ -13,29 +13,17 @@ services:
       POSTGRES_USERNAME: postgres
       POSTGRES_HOST: ffc-demo-user-postgres
       POSTGRES_PORT: 5432
-    networks:
-      - ffc-demo-users
-    ports:
-      - "3002:3002"
     volumes:
       - node_modules:/usr/src/app/node_modules
       - .:/usr/src/app
 
   ffc-demo-user-postgres:
-    container_name: ffc-demo-user-postgres
+    container_name: ffc-demo-user-postgres${BUILD_NUMBER:-local}
     image: postgres:11.4-alpine
     environment:
       POSTGRES_DB: ffc_demo_users
       POSTGRES_PASSWORD: ppp
       POSTGRES_USERNAME: postgres
-    networks:
-      - ffc-demo-users
-    volumes:
-      - postgres_data:/var/lib/postgresql/data
-
-networks:
-  ffc-demo-users: {}
 
 volumes:
   node_modules: {}
-  postgres_data: {}

```

Comment:
The `postgres_data` volume is declared in `docker-compose.link.yaml` as well as `docker-compose.override.yaml`. Is it worth moving back to the base file to reduce duplication?

I think there is value in creating volumes when running tests so that the data can be inspected after a test fails.

Reply:
I'd prefer the tests not to use persistent volumes to avoid issues with caching, i.e. a change in database name or username between branches preventing tests from running.

Reply:
The test script wipes out volumes before each run and after a successful run so that wouldn't be a problem anymore.

Reply:
I'm happy to leave it for now if you'd prefer.

Reply:
fair enough, I've pushed the volume down to the base file and removed it from the override and link files.